docs/doc/index.md

```diff
@@ -2,17 +2,16 @@
 
 The Reference System server provides a set of services necessary to build Copernicus processing workflows. All services are subject to access control.
 
-RS-Server offers a catalog of Sentinel products compatible with the STAC (SpatioTemporal Asset Catalog) standard, as well as functions to access data from AUXIP and CADIP stations.
+STAC is everywhere with RS-Server. RS-Server offers a catalog of Sentinel products compatible with the STAC (SpatioTemporal Asset Catalog) standard, but also on the next release the staging function to retrieve AUXIP and CADIP data will be provided with a STAC interface.
+
+![STAC](images/STAC-2.jpg)
```

Comment:
It's because the images folder is not in the same folder of index.md document. I guess you need to change from :
`![STAC](images/STAC-2.jpg)`
to :
`![STAC](../images/STAC-2.jpg)`

Reply:
I have fix the issue. Can you please merge ?